.circleci/scripts/tag-rc-if-necessary.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -x
+
+PREVRC=$(git for-each-ref --sort=-creatordate --format="%(refname:short)|%(creatordate:short)" refs/tags/* | grep "0-rc|" | head -n 1)
+
+TAGDATE=$(echo $PREVRC | cut -d"|" -f2)
+taggedDate=$(date --date "$TAGDATE" +"%s")
+threeWeekDate=$(date --date "21 days ago" +"%s")
```

Comment:
Looks like it can be fragile. If script three weeks ago finished its work one minute later, `[[ ${taggedDate} -lt ${threeWeekDate} ]]` can be false.
Could we change it to 20 days?

Reply:
@vitaliyreznikov I used %(creatordate:short) a little bit above, which only has the date. So TAGDATE would be 12:00AM UTC of the day the last rc was tagged. Do you still think it's necessary to change to 20 days?

Reply:
I don't mind the 20 days thing either. It's a little safety net and might catch some silly cases like daylight saving shenanigans adjusting the crontab date slightly.

Reply:
Ok i'll make this change.

Reply:
Done. Also just use unix timestamp since we have an extra day now.